doc/manpages/faq.asciidoc

```diff
@@ -133,6 +133,16 @@ The shell scope allows users to spawn any interpreter they want, for a minimal
 cost in terms of performance, it is therefore the reason why it's the only
 one available by default.
 
+What shell is used to expand `%sh{}` scopes ?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The server expands shell scopes using `/bin/sh` (which is, in most
```

Comment:
It used to be that, then we discovered that `/bin/sh` was not stricly guaranteed to be posix, that the only guaranteed way to get a posix tool on a posix system was through that `confstr` call. I think that changed around a year ago.